docs/build/user-consent.md

```diff
@@ -0,0 +1,336 @@
+---
+sidebar_label: User consent
+sidebar_position: 5.5
+hide_table_of_contents: false
+---
+
+import Tabs from "@theme/Tabs";
+import TabItem from "@theme/TabItem";
+import requeststab from '/docs/build/img/requests-tab.png';
+import messagestab from '/docs/build/img/messages-tab.png';
+import consentlogic from '/docs/build/img/consent-state-logic.png';
+
+# Request and respect user consent
+
+![Feature status](https://img.shields.io/badge/Feature_status-Alpha-orange)
+
+:::caution This feature is in **alpha** status
+
+This feature is in **alpha** status and ready for you to start experimenting with. However, we do not recommend using alpha features in production apps. Expect frequent changes as we iterate based on feedback. Want to provide feedback? Comment on [Proposal: Portable consent state for v2 SDKs](https://github.com/orgs/xmtp/discussions/49). 
+
+:::
+
+The user consent feature enables your app to request and respect user consent preferences. With this feature, a blockchain account address registered on the XMTP network can have one of three consent preference values:
+
+- Unknown
+- Allowed
+- Denied
+
+These values express a user's consent preferences for a contact. These consent preferences are stored in a consent list on the XMTP network. The consent list is accessible by all apps.
+
+## Understand user consent preferences
+
+Here are some of the ways user consent preferences are set:
+
+### Unknown
+
+Conversation created in an app on an SDK version **with** user consent support:
+
+- For a new conversation that a peer contact wants to start with a user, the consent preference is set to `unknown`.
+
+Conversation created in an app on an SDK version **without** user consent support:
+
+- For all conversations with any peer contact, the consent preference is set to `unknown`.
+
+### Allowed
+
+Conversation created in an app on an SDK version **with** user consent support:
+
+- For a new conversation that a user created with a peer contact, the SDK sets the consent preference to `allowed`.  
+
+  The user’s creation of the conversation with the contact is considered consent.
+
+- For an existing conversation created by a peer contact that hasn’t had its consent preference updated on the network (`unknown`) and that the user responds to, the network will update the consent preference to `allowed`.  
```

Comment:
@nplasterer May I ask you - we updated this to say "the network will update..." 

In other places we say that the "SDK" sets the preference. 

Should they all say the "network" or the "SDK" - or is this one unique in that the network will do it? 😀 TY!

Reply:
i think "the SDKs" is more fitting since they are making the network calls to allow consent

Reply:
DONE. <3